tools/fmt/writer.rs

```diff
@@ -0,0 +1,37 @@
+// Copyright © SixtyFPS GmbH <info@sixtyfps.io>
+// SPDX-License-Identifier: (GPL-3.0-only OR LicenseRef-SixtyFPS-commercial)
```

Comment:
The CI system is checking the license header on every file, and that is the old one.

```suggestion
// Copyright © SixtyFPS GmbH <info@slint-ui.com>
// SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-Slint-commercial
```